README.md

```diff
@@ -258,6 +273,7 @@ Arguments:
 - name: The name of the publication
 - collection: A function that returns the collection to publish (or just a collection, if it's insecure).
 - selector: (Optional) A function that returns mongo selector that will limit the results published (or just the selector).
+- settings: (Optional) A object with settings on server side's publish function. (Details see below)
```

Comment:
After some more thoughts i would like to prevent it on server side, since regex might turn into a weapon.
Maybe a client side option as second addition.
However: server side must be overruling whatever client wants to not expose something, that should be hidden.